packages/release/prepare-bottlerocket-netdog.service

```
@@ -0,0 +1,20 @@
+[Unit]
+Description=Prepare Netdog OverlayFS Directories in /.bottlerocket
```

For consistency with the other combinations of "mount units" and "helpers that create some directories after the mount":
```suggestion
Description=Prepare Private Directory (/.bottlerocket)
```

I would also rename the service to `prepare-bottlerocket.service` since we might add other preparations later.

---

I updated to be `Prepare Netdog Directory (/.bottlerocket/netdog)`

Let me know if that makes sense.